src/common/addlayers/RegistryLayersDirective.js

```diff
@@ -269,6 +278,15 @@
 
             // load saved registry layers if they exist
             scope.addRegistryLayersFromSavedMap();
+
+            function cartLayer(layerConfig) {
```

Comment:
@milafrerichs this function just add the to the map preview the layers that are inside the cart.